README.md

```diff
@@ -1,2 +1,76 @@
-# intergalactic-most-wanted-list
-A database for malware checksums for use with the WallE malware cleanup script
+# Intergalactic Most Wanted List
+A database for malware checksums for use with the [WallE](https://github.com/usegalaxy-eu/WallE) malware cleanup script.
+
+This database in form of a yaml file stores malware checksums and its metadata
+schema:
+~~~yaml
+class:
+  program:
+    version:
+      severity: [high, medium, low]
+      description: "optional info"
+      checksums:
+        crc32: <checksum crc32, gzip algorithm, integer representation>
+        sha1: <checksum sha1, hex representation>
+~~~
+Contributions are always welcome. Here is a more detailed explanation how the schema works.
+### Class
+What purpose does it serve, e.g. mining, (port-)scanner, multi-purpose...  
+
+**Not** a `class` would be `nmap`, it is a specific tool for scanning and would be a `program` under the `scanning` class.
+
+### Program
+I did not come up with a better title for now. (Refused to call it 'tool')  
+Feel free to rename this.  
+
+A `program` is every single unit of malware. In most cases it would be an executable file. But it can be a script, a binary, a package or a zip archive.  
+Think of everything that you would version.
```

Comment:
no the intention was to explain what kind of entity to put there and that everything that can be versioned is a "program". `miniconda3` already contains a version and should not be the name for a program, for instance.

Reply:
```suggestion
Ideally, a version corresponds to the version of your program. In case it does not have a version of you don't know, provide an arbitrary version for example `1`, `2` so we can distinguish different checksum from the `program`.
```

Reply:
```suggestion
```